app/views/page/show.html.erb

```diff
@@ -56,6 +55,8 @@
 
           <%# Accordion %>
         <% when 'PageAccordionComponent' %>
+          <% accordion_ctr += 1 %>
+          <div id="accordion_anchor_<%= accordion_ctr %>"/>
```

Comment:
Do we need to add a `div` here? Can't we just use the container div on the next line? Also, would it make sense to add a column in the DB called `anchor_link_name`, or something like that, which would allow the user to create their own anchor name, and then we can just `.parameterize` it? I'd like to know your thoughts on that.

Reply:
I think I see what you mean..  add the column (anchor_link_name) to the page_components table right?   So when we save the page in PB we would just create a guid for the anchor_link_name and then embed that in the PB show page?  Only problem I see with that it may be harder for the content creator to create the anchor links in PB instead of just using a convention like "accordion_anchor_1"..etc or "map_anchor_1" ...etc.  ?  Lets talk through this.

Reply:
Moved anchor_tag id to div container instead of above div container.. that works.